src/Comments.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { fetchComments } from "./apis/api";
+
+function Comments() {
+  const [comments, setComments] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
```

Comment:
I'm craving a loading state here for fetching of the comments themselves, with a relevant message for the user.